site/src/pages/docs/client-retry.mdx

```diff
@@ -71,7 +71,7 @@ new RetryRule() {
             }
         }
 
-        if (ctx.log().responseHeaders().status() == HttpStatus.TOO_MANY_REQUESTS) {
+        if (ctx.log().ensureAvailable(RequestLogProperty.RESPONSE_HEADERS).responseHeaders().status() == HttpStatus.TOO_MANY_REQUESTS) {
```

Comment:
could you break the line, please?
or extract the status to a variable so that the line doesn't get too long?

Reply:
introduced a variable.